xnat_tools/bids_utils.py

```diff
@@ -379,10 +379,35 @@ def scan_contains_dicom(connection, host, session, scanid):
     return True
 
 
+def download_resources(connection, host, session, bids_session_dir):
+    r = get(
+        connection,
+        host + "/data/experiments/%s/files" % session,
+        params={"format": "json"},
+    )
+
+    # Build dictionary of format { filename: (pathURI, collectionType) } for every resource
+    resourceFileDict = {
+        resource["Name"]: ({"URI": host + resource["URI"]}, resource["collection"])
+        for resource in r.json()["ResultSet"]["Result"]
+    }
+
+    resourceFileList = list(resourceFileDict.items())
+
+    # Download Resources
+    for name, resourceDetails in resourceFileList:
+        _logger.info("Downloading files")
```

Comment:
I'd suggest changing this to `_logger.info("Downloading file: "+name)` so they can see what files they're getting